content/post/blacklivesmatter.md

```diff
@@ -0,0 +1,66 @@
+---
+author: Lewis Denham-Parry
+categories:
+- meetup
+- community
+date: 2020-06-13T14:00:00Z
+draft: false
+slug: blacklivesmatter
+title: Cloud Native Wales supports Black Lives Matter
+---
+
+This blog post will continue to be updated via contributions to the website [repo](https://github.com/CloudNativeWales/Hugo), if you would like to contribute to the post please raise a pull request.  If you don't know what a pull request is please contact [Lewis](mailto:lewis@cloudnativewales.io) who will be more than happy to help you.
+
+---
+
+This part of the post is by [Lewis Denham-Parry](https://twitter.com/denhamparry).  I'm sharing my experience and want to continue to learn and support others.  Please DM or contact me via [email](mailto:lewis@cloudnativewales.io) if you'd like to contribute or discuss any points about regarding this post or BLM.
+
+- I've been following BLM the day after George Floyd's murder
+- Remember seeing George Floyd in the news
+  - Was in Minnesota this time last year
+  - Sadly this was another story that I felt like I've seen 100's of times and desensitised to it
+  - I've never watched the video, spent hours reading about it but never watched it
+  - I saw the outrage from my friends who live there and then the riots that started
+- Been learning to take a step back
+  - Had feedback that I get too involved with everything
+  - Told I keep trying to solve problems
+  - I wanted to hear other peoples voices to understand what was going on
+  - I've never seen colour, race, religion, gender, sexuality as an issue
+  - The understanding my personal opinion solves the problem
+  - This post has started a lot with points about me
+  - Still trying to figure out if it was wrong of me to wait to try to understand what has and is going on
+- This isn't an American problem
+  - Easy to think that this is someone elses problem
+  - Harder to look and realise that what you thought was equal isn't
+  - Even harder to want to change the normal to make it fair
+  - Do we have these issues in the UK and in Wales
+- Reached out to friends who would be deemed to be in a minority
+  - Listening to others, I now have a better understanding that this isn't about now, this is about the past and the future
+  - Have found out that people expect to be treated differently
+  - Was it wrong that I thought it'd be racist of me to presume that my friend would get pulled over for driving a nice car in Cardiff?
+  - Was it wrong for me to think I was being racist for being yet another privileged friend to ring and ask them how they're doing and how it affects them?
+- How do we solve this?
+  - Don't know how we solve this, but I know we can make this better each and every day
+  - Promote diversity in your workspace, at home and everywhere you look
```

Comment:
This bullet point is a little vague. I found [this article](https://mashable.com/article/black-lives-matter-tweets-tech-leaders/?europe=true) about BLM and the tech industry. Some of its points are a bit more concrete and could be included in this section (with attribution)